src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -21,7 +25,7 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.0-beta3" />

```

Comment:
Before this can be merged, we'll also need to update this package reference to a version of Microsoft.Azure.WebJobs that contains your new attribute.